app/lib/report_helper.rb

```diff
@@ -34,7 +34,9 @@ module ReportHelper
     threshold_alignmentlength: 0.0,
     threshold_neglogevalue:    0.0,
     threshold_aggregatescore:  0.0,
-    excluded_categories: 'None'
+    excluded_categories: 'None',
+    selected_genus: 'None',
+    disable_filters: 0
```

Comment:
@cdebourcy You just need to add `&disable_filters=1` for your CSV download.

Reply:
thanks!